docs/interaction_based_testing.adoc

```diff
@@ -942,18 +933,33 @@ real subscribers to the mock object. The mock object's instance isn't ever passe
 it is only used to describe the interaction.
 
 NOTE: A global mock can only be created for a class type. It effectively replaces
-all instances of that type for the duration of the feature method.
+all instances of that type for the duration from mock creation up until the end of the feature method.
+
+CAUTION: The order of declaration global mocks is relevant.
+         The `GroovySpy(global:true)` must come before all creations of new mocked/spied objects.
```

Comment:
I don't think this is entirely clear, is that just for instances of the spied upon class or all other mocks?

Reply:
I have added a bit more text to clarify this.